docusaurus/docs/guides/securing-apis/aks-api-with-nginx-ziti-module.md

```diff
@@ -0,0 +1,369 @@
+---
+title: AKS API with NGINX Ziti Module
+---
+
+## Prerequisites
+
+  - Azure Subscription, Resource Group and [Azure Cli](https://learn.microsoft.com/en-us/cli/azure/)
+  - [OpenZiti Nginx Module Repo](https://github.com/openziti/ngx_ziti_module)
+  - [Terraform](https://developer.hashicorp.com/terraform/downloads)
+  - Openziti Network
+  - [Desktop Tunneler](https://docs.openziti.io/docs/reference/tunnelers/)
+---
+## Architecture:
+- Before
+![](../../../static/img/nginx-aks-before.svg)

```

Comment:
I'm not sure which way is best. It's convenient to have the images in the same dir as the Markdown, and I don't think it's necessary to organize them under `/static`. It can be a little confusing to mentally map the static path back to the docs path. Do you prefer to put them under static or all the same to you?

Reply:
I am ok either way

Reply:
OK, please organize the images in the same dir as the Markdown that uses them.

Reply:
I added img folder in the directory and added these files there.